### ISSUE TITLE
Fix database connection on prod and optimize envfiles

### ISSUE THREAD

**Issue by hwakabh** [hwakabh/bennu-official#76]
# AsIs
Application would be now running on GKE with #61 (and #62) , but when connecting to mysql database container, we got errors with connection refused, as attached.
This is because of `settings.py` would be still for local environment, so need to update as k8s service name.

<img width="938" alt="Screenshot 2024-01-09 at 16 12 13" src="https://github.com/hwakabh/bennu-official/assets/25563897/745f81f0-1aee-4e33-8dfb-9a7e792770ce">

# ToDo
- Update settings.py for production, or separate with local_settings.py
- Create envfiles for local and prod

# (Optional) Acceptance criteria
We can access `http://${external_ip_django}/discography`, where routing requires to connect mysql database.

**Comment by hwakabh** [comment]
After the merging associated PR, the connection errors to MySQL container would be cleared, but loading of staticfiles would be failed in Django container after executing migration on GKE.
```shell
% kubectl -n bennuhp exec -it django-7d66c74b97-nsvlq -- /bin/bash
cnb@django-7d66c74b97-nsvlq:/workspace$ 
cnb@django-7d66c74b97-nsvlq:/workspace$ ls -al /cnb/process/
total 8
drwxr-xr-x 2 root root 4096 Jan  1  1980 .
drwxr-xr-x 1 root root 4096 Jan  1  1980 ..
lrwxrwxrwx 1 root root   23 Jan  1  1980 createuser -> /cnb/lifecycle/launcher
lrwxrwxrwx 1 root root   23 Jan  1  1980 migrate -> /cnb/lifecycle/launcher
lrwxrwxrwx 1 root root   23 Jan  1  1980 web -> /cnb/lifecycle/launcher
cnb@django-7d66c74b97-nsvlq:/workspace$ 
cnb@django-7d66c74b97-nsvlq:/workspace$ /cnb/process/migrate 
Operations to perform:
  Apply all migrations: admin, auth, bennuhp, contenttypes, sessions
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying auth.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying bennuhp.0001_initial... OK
  Applying bennuhp.0002_liveschedule_movie_music_delete_blogpost... OK
  Applying bennuhp.0003_remove_liveschedule_id_remove_movie_id_and_more... OK
  Applying bennuhp.0004_alter_liveschedule_uuid_alter_movie_uuid_and_more... OK
  Applying bennuhp.0005_movie_image_path_music_image_path... OK
  Applying sessions.0001_initial... OK

151 static files copied to '/workspace/assets', 421 post-processed.
```

<img width="584" alt="Screenshot 2024-01-10 at 15 03 13" src="https://github.com/hwakabh/bennu-official/assets/25563897/23f81cc9-012a-49c7-ab9a-f7560e6323a1">

This is because, `STATIC_URL` would be not matched in the container, hence we can see the logs with django container:
```log
% kubectl -n bennuhp logs django-598679498-pfr77 -f
[2024-01-10 05:57:20 +0000] [1] [INFO] Starting gunicorn 21.2.0
[2024-01-10 05:57:20 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2024-01-10 05:57:20 +0000] [1] [INFO] Using worker: sync
[2024-01-10 05:57:20 +0000] [9] [INFO] Booting worker with pid: 9
[2024-01-10 05:57:20 +0000] [10] [INFO] Booting worker with pid: 10
# (...)
10.128.0.52 - - [10/Jan/2024:15:01:26 +0900] "GET /discography/ HTTP/1.1" 200 4220 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
10.128.0.52 - - [10/Jan/2024:15:01:26 +0900] "GET /assets/bennuhp/css/bootstrap.d39b3b19bd05.css HTTP/1.1" 404 3756 "http://34.29.188.169/discography/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
10.128.0.52 - - [10/Jan/2024:15:01:26 +0900] "GET /assets/bennuhp/css/main.f6bf0188774d.css HTTP/1.1" 404 3756 "http://34.29.188.169/discography/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
```

**Comment by hwakabh** [comment]
As staticfile errors would be fixed in #78 and the acceptance of this issue would be cleared, closed the issue for now.
